lib/commands/context.js

```diff
@@ -169,6 +169,15 @@ extensions.getContextsAndViews = async function (useUrl = true) {
   return ctxs;
 };
 
+extensions.getNewRemoteDebugger = async function () {
```

Comment:
should it be async?

Reply:
Please see PR description (https://github.com/appium/appium-ios-driver/pull/269#issue-174448041).